Makefile

```diff
@@ -38,77 +38,53 @@ list:
 %.osm.pbf:
 	@echo "Downloading $@ from BBBike.";
 	@echo "\n\nConsider donating to BBBike to help cover hosting! https://extract.bbbike.org/community.html\n\n"
-	wget -U headway/1.0 -O $@ "https://download.bbbike.org/osm/bbbike/$(basename $(basename $@))/$@" || rm $@
+	wget -U headway/1.0 -O $@ "https://download.bbbike.org/osm/bbbike/$(notdir $(basename $(basename $@)))/$(notdir $@)" || rm $@
 
 %.bbox:
 	@echo "Extracting bounding box for $(basename $@)"
 	grep "$(basename $@):" gtfs/bboxes.csv > $@
 	perl -i.bak -pe 's/$(basename $@)://' $@
 
 %.mbtiles: %.osm.pbf
-	@echo "Building MBTiles $(basename $@)"
-	mkdir -p ./.tmp_mbtiles
-	cp $(basename $@).osm.pbf ./.tmp_mbtiles/data.osm.pbf
-	docker volume create headway_mbtiles_build || echo "Volume already exists"
+	@echo "Building MBTiles $(notdir $(basename $@))"
 	docker build ./mbtiles/bootstrap --tag headway_mbtiles_bootstrap
```

Comment:
There's another easy win here, we could just untar sources.tar into the data directory. The only reason it was a container before was that gave it access to a volume, but if the volume is a host directory the makefile can do this and we can remove the bootstrap image.